src/frontend/types/AppData.ts

```diff
@@ -23,7 +23,7 @@ export interface AppData {
   documents?: Document[];
   resource?: any;
   modelName: modelName.VIDEOS | modelName.DOCUMENTS;
-  appName: string;
+  appName?: 'bbb';
```

Comment:
Do you think we could get available app names from the settings here ? https://github.com/openfun/marsha/blob/master/src/frontend/settings.ts#L9

Reply:
yep, I think it will only require to declare app name values in an enum and use the values of this enum in the type and in the settings
I will take a look